datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaBasedConnectorTask.java

```diff
@@ -1099,4 +1103,9 @@ public static String getKafkaGroupId(DatastreamTask task, GroupIdConstructor gro
   public String getConsumerAutoOffsetResetConfig() {
     return _consumerProps.getProperty(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "");
   }
+
+  @VisibleForTesting
+  public long getStoppedLatchCount() {
```

Comment:
is `public` required for this method?

Reply:
yes I am using this method in TestKafkaMirrorMakerConnectorTask.java

Reply:
But that test is in the same package. This should not be public, should just be default (package) scope:

`long getStoppedLatchCount() {`